packages/itwin/reports-config-widget/public/locales/en-PSEUDO/ReportsConfigWidget.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
you shouldnt need to commit this file, we also never deliver the psuedo localized strings

Reply:
Removed